astlib/location.mli

```diff
@@ -52,6 +52,10 @@ val set_input_lexbuf : Lexing.lexbuf option -> unit
 val none : t
 (** An arbitrary value of type [t]; describes an empty ghost range. *)
 
+val dummy_pos : Lexing.position
```

Comment:
I'm not sure we should expose this as Astlib is supposed to represent the minimum stable API we'd need from the compiler and it seems that something like `in_file` would be enough here wouldn't it?